### PR TITLE
Profile stuff and navbar update

### PR DESCRIPTION
**Pull request by bartorl**
Linking from navbar to settings and profile, settings and profile created

### CODE REVIEW

File: views/partials/navbarloggedin.ejs [hunk 1]
```diff
@@ -53,8 +53,8 @@
                   <%= user.username %>
                 </a>
                 <ul class="dropdown-menu me-1 dropdown-menu-lg-end">
-                  <li><a class="dropdown-item" href="#">Profile</a></li>
-                  <li><a class="dropdown-item" href="#">User settings</a></li>
+                  <li><a class="dropdown-item" href="/profile">Profile</a></li>
```
Comment (Direckton) [inline views/partials/navbarloggedin.ejs:56]: The path to res.render is  'profileSettings/profile', verify with JK the path is correct
https://github.com/MasterCoookie/bezsciemy/blob/master/controllers/profileSettingsController.js#L2

File: views/partials/navbarloggedin.ejs [hunk 2]
```diff
@@ -53,8 +53,8 @@
...
+                  <li><a class="dropdown-item" href="/settings">User settings</a></li>
```
Comment (Direckton) [inline views/partials/navbarloggedin.ejs:57]: same here

**Review by MasterCoookie — APPROVED** [review]

File: views/profileSettings/settings.ejs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<%- include('../partials/head.ejs') %>
+    <div>
+        <h3 class="text">Yout current email:</h3>
```
Comment (tomazaw) [inline views/profileSettings/settings.ejs]: typo

**Review by tomazaw — CHANGES_REQUESTED** [review]

File: views/profileSettings/settings.ejs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <form name="emailform">
+            <label for="newemail" class="text">Change your email here:</label><br>
+            <textarea id="newemail" name="newemail"
```
Comment (macibol775) [inline views/profileSettings/settings.ejs]: ZWROT

File: views/profileSettings/settings.ejs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            <label for="newemail" class="text">Change your email here:</label><br>
+            <textarea id="newemail" name="newemail" 
+            placeholder="Write your new email here">
+            </textarea><br>
+            <button type="submit" id="sendnewemail">Change email</button>
+        </form>
+        <form name="passwordform">
+            <h3 class="text">Change your password</h3>
+            <label for="oldpassword" class="text">Write your current password here:</label><br>
+            <textarea id="oldpassword" name="oldpassword"
```
Comment (macibol775) [inline views/profileSettings/settings.ejs]: ZWROT

File: views/profileSettings/settings.ejs [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            <label for="oldpassword" class="text">Write your current password here:</label><br>
+            <textarea id="oldpassword" name="oldpassword" 
+            placeholder="Write your current password here">
+            </textarea><br>
+            <label for="newpassword" class="text">Write your new password here:</label><br>
+            <textarea id="newpassword" name="newpassword" 
+            placeholder="Write your new password here">
+            </textarea><br>
+            <label for="repeatpassword" class="text">Repeat your new password:</label><br>
+            <textarea id="repeatpassword" name="repeatpassword"
```
Comment (macibol775) [inline views/profileSettings/settings.ejs]: ZWROT

**Review by macibol775 — CHANGES_REQUESTED** [review]
textarea na hasło ?!?!?!?!?!?

**Review by tomazaw — APPROVED** [review]

**Review by Direckton — APPROVED** [review]

File: views/partials/navbarloggedin.ejs (hunk 1 above)
Comment (bartorl) [reply views/partials/navbarloggedin.ejs:56]: I tested it and it works

**Review by macibol775 — COMMENTED** [review]
Write? better type / input

używajjakiejgośstandarduaniewszystkomałymiliteramiok

File: views/partials/navbarloggedin.ejs (hunk 1 above)
Comment (MasterCoookie) [reply views/partials/navbarloggedin.ejs:56]: its ok

**Review by macibol775 — COMMENTED** [review]
wiadomo co

**Review by macibol775 — APPROVED** [review]
ok

**Pull request merged**